ui/raidboss/data/05-shb/dungeon/paglthan.js

```diff
@@ -134,6 +134,42 @@ export default {
         },
       },
     },
+    {
+      id: 'Paglthan Lunar Flare Reset',
+      netRegex: NetRegexes.ability({ id: '5B49', source: 'Lunar Bahamut', capture: false }),
+      netRegexDe: NetRegexes.ability({ id: '5B49', source: 'Luna-Bahamut', capture: false }),
+      netRegexFr: NetRegexes.ability({ id: '5B49', source: 'Luna-Bahamut', capture: false }),
+      netRegexJa: NetRegexes.ability({ id: '5B49', source: 'ルナバハムート', capture: false }),
+      run: (data) => data.lunarFlares = 0,
+    },
+    {
+      // Get middle is 4x5B4A and 4x5B4B, get outside is 5x5B4A
+      id: 'Paglthan Lunar Flare',
+      netRegex: NetRegexes.startsUsing({ id: '5B4A', source: 'Lunar Bahamut', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '5B4A', source: 'Luna-Bahamut', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '5B4A', source: 'Luna-Bahamut', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '5B4A', source: 'ルナバハムート', capture: false }),
+      preRun: (data) => data.lunarFlares = (data.lunarFlares || 0) + 1,
+      alertText: (data, _, output) => {
+        if (data.lunarFlares === 4)
+          return output.getMiddle();
+        if (data.lunarFlares === 5)
+          return output.getOutsideBetweenCircles();
+      },
```

Comment:
Will this always call get middle on the get outside version? Do you need a separate collect trigger and then this one gets a delay?

Reply:
Good point, I have an idea on a simple fix.